Sources/PostgresNIO/Utilities/PostgresJSONCoder.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+public struct PostgresJSONCoder {
+    public static var global: PostgresJSONCoder = .init()
```

Comment:
Are these thread safe? I'm pretty sure they're not on Linux

Reply:
This code is almost a copy/paste from https://github.com/vapor/vapor/blob/master/Sources/Vapor/Content/ContentConfiguration.swift 

So if it isn't thread safe, then vapor isn't thread safe either.

Reply:
`JSONEncoder` and `JSONDecoder` are just wrapper types that initialize `_JSONEncoder` and `_JSONDecoder` respectively on each call.